### PR TITLE
[Homework #19] - React Hooks by Stanisław Królikiewicz

### PR DESCRIPTION
**Pull request by stkrolikiewicz**

### CODE REVIEW

File: homeworks/stanislaw-krolikiewicz/19-react-hooks/src/pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {AddTaskInput, Error, TasksList} from '@/components';
+
+export default function Home() {
```
Comment (VitalikMelnyk) [inline homeworks/stanislaw-krolikiewicz/19-react-hooks/src/pages/index.tsx:3]: Fix responsive design.
Screenshot 2023-10-16 at 11 29 19

File: homeworks/stanislaw-krolikiewicz/19-react-hooks/src/components/Task/Component.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+export default ({task}: Props) => {
+  const {tasks, setError, updateTask, deleteTask} = useContext(TasksContext);
+  const [name, setName] = useState(task.name);
+  const refInput = useRef<HTMLInputElement>(null);
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    setName(e.target.value);
+  };
+  const handleUpdate = () => {
```
Comment (VitalikMelnyk) [inline homeworks/stanislaw-krolikiewicz/19-react-hooks/src/components/Task/Component.tsx:19]: Fix edit functionality. When I click edit button, I have got an error with "Task already exists"
Screenshot 2023-10-16 at 11 29 01

**Review by VitalikMelnyk — CHANGES_REQUESTED** [review]
Hello @stkrolikiewicz!
I have checked your PR and left some comments.

File: homeworks/stanislaw-krolikiewicz/19-react-hooks/src/pages/index.tsx (hunk 1 above)
Comment (stkrolikiewicz) [reply homeworks/stanislaw-krolikiewicz/19-react-hooks/src/pages/index.tsx:3]: @VitalikMelnyk I've provided the changes.

**Review by VitalikMelnyk — APPROVED** [review]
Approved.

**Pull request merged**